.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+    "javascript.validate.enable": false,
+    "flow.useNPMPackagedFlow": true,
+    "flow.runOnEdit": true
+}
```

Comment:
please fix your editor to respect [.editorconfig](http://editorconfig.org/#download)